### DISCUSSION TITLE
[BUG] Invalid device info "WDR96WXB" (hard disk serial number)

### DISCUSSION THREAD

**Discussion by Yogesh7920** [dashingsoft/pyarmor#1392] [Q&A]
First I got my hard disk serial number by running the command
```
wmic diskdrive get serialnumber   
```

Then, I ran the following 
```
$ pyarmor gen -O "../nse_dist" -b WDR96WXB app.py
```
I got the following error
ERROR    invalid device info "WDR96WXB"

**Comment by jondy** [comment]
So what's the output of `pyarmor-7 hdinfo`?
